netmiko/cisco/cisco_xr.py

```diff
@@ -166,11 +213,13 @@ def exit_config_mode(self, exit_config: str = "end", pattern: str = "") -> str:
             # Read until we detect either an Uncommitted change or the end prompt
             if not re.search(r"(Uncommitted|#$)", output):
                 output += self.read_until_pattern(pattern=r"(Uncommitted|#$)")
-            if "Uncommitted changes found" in output:
+            if "Uncommitted" in output:
                 self.write_channel(self.normalize_cmd("no\n"))
                 output += self.read_until_pattern(pattern=r"[>#]")
             if not re.search(pattern, output, flags=re.M):
```

Comment:
can you check why is this needed again ?

Reply:
to search for a user provided pattern. This function has no custom additions by us.

Reply:
This if condition exists from the origin code with no custom addition by us. I would keep it there to keep more consistent with the open source code